### PR TITLE
Build ensembler on master.

### PR DESCRIPTION
**Pull request by ashwinath**
Currently, Github actions is not publishing to master. This will allow ensemblers to build on master.

### CODE REVIEW

File: .github/workflows/engines-batch-ensembler.yaml [hunk 1]
```diff
@@ -60,6 +60,8 @@ jobs:
       ) || (
         github.event_name == 'push' &&
         github.event.pull_request.head.repo.full_name == github.repository
+      ) || (
+        github.ref == 'refs/heads/master'
```
Comment (romanwozniak) [inline .github/workflows/engines-batch-ensembler.yaml:64]: This entire `if` condition should be reviewed and if necessary – cleaned (I think it was copied from somewhere). But in the context of this PR – LGTM

**Review by romanwozniak — APPROVED** [review]

File: .github/workflows/engines-batch-ensembler.yaml (hunk 1 above)
Comment (ashwinath) [reply .github/workflows/engines-batch-ensembler.yaml:64]: yeah this master part was copied.

**Comment by ashwinath** [comment]
The E2E tests were failing (strange as it was working yesterday). I'm not sure what was going on here: https://github.com/gojek/turing/runs/2950046686?check_suite_focus=true#step:4:224 but it seems like the kube-scheduler had some permissions issue and could not list/add resources. Upgrading the Kind and Kubernetes version seemed to help.

To fix it, I upgraded the Kind CLI version and the Kubernetes patch version (1.17.11 -> 1.17.17).

**Review by romanwozniak — APPROVED** [review]
I'm not sure if the fix is related to the upgrade of infra dependencies, but it doesn't hurt upgrading anyway, so lgtm

**Pull request merged**